src/commands.rs

```diff
@@ -874,7 +890,8 @@ mod test {
                             "--change_descriptor", "wpkh(tpubDEnoLuPdBep9bzw5LoGYpsxUQYheRQ9gcgrJhJEcdKFB9cWQRyYmkCyRoTqeD4tJYiVVgt6A3rN6rWn9RYhR9sBsGxji29LYWHuKKbdb1ev/1/*)",
                             "create_tx", "--to", "n2Z3YNXtceeJhFkTknVaNjT1mnCGWesykJ:123456","mjDZ34icH4V2k9GmC8niCrhzVuR3z8Mgkf:78910",
                             "--utxos","87345e46bfd702d24d54890cc094d08a005f773b27c8f965dfe0eb1e23eef88e:1",
-                            "--utxos","87345e46bfd702d24d54890cc094d08a005f773b27c8f965dfe0eb1e23eef88e:2"];
+                            "--utxos","87345e46bfd702d24d54890cc094d08a005f773b27c8f965dfe0eb1e23eef88e:2",
+                            "--add_string","Hello BDK" ];
```

Comment:
We can add `add_string` into the same test and ensure that both are being parsed..

Reply:
> utACK [29c0686](https://github.com/bitcoindevkit/bdk-cli/commit/29c068683c388ceafea1ad4f4d0afaf4311c64a0)
> 
> Nice one.. This will be really useful.. Below are just few nits..
> 
> Will you be willing to write a demo test for this after #102 lands?? We are going to have integration testing after 102, and that can be good place to document demonstrations of bdk-cli for this kinda stuffs too..

Hi Raj, I'm willing to write a demo test. Also, I've added `add_data` into the same test. You mentioned adding to the test `add_string` but I think you meant `add_data`.  Thank you for the review and suggestions you mentioned. 😄

Reply:
Oh sorry yup that's what I meant.. Thanks for the quick update..